docs/docs/http-api.md

```diff
@@ -0,0 +1,69 @@
+# HTTP API reference
+
+Drand provides a JSON HTTP interface that clients can use to fetch randomness from the drand network. If you're using drand in an application, it may be easier and _more secure_ to use one of the [client libraries](/docs/client-libraries/), which will also perform _verification_ of randomness rounds and add additional features like failover, racing, aggregation, and caching.
+
+All that's required is the address of the HTTP interface and way to fetch from HTTP, e.g. `curl`.
+
+The public [League of Entropy](https://blog.cloudflare.com/league-of-entropy/) HTTP APIs are available at:
+
+- [https://drand.cloudflare.com](https://drand.cloudflare.com)
+- [https://drand.protocol.ai:8081](https://drand.protocol.ai:8081)
+- [https://ln.soc1024.com:8888](https://ln.soc1024.com:8888)
+- [https://drand.nikkolasg.xyz:4444](https://drand.nikkolasg.xyz:4444)
+- [https://drand2.kudelskisecurity.com](https://drand2.kudelskisecurity.com)
+
+The chain hash for the League of Entropy drand group is `TODO-LOE-CHAIN-HASH`
+
+## `/info`
+
+Retrieves the randomness chain information. It returns a JSON object with the following structure:
+
+```json
+{
+  "public_key": "aaddd53d2c92454b698c52495990162bc999778a32fd570dad2ef3de2915a5b397d80ec5508919e84cd10944955b7318",
+  "period": 10,
+  "genesis_time": 1592226590,
+  "hash": "c599c267a0dd386606f7d6132da8327d57e1004760897c9dd4fb8495c29942b2"
+}
+```
+
+- `public_key` is the distributed public key of the drand group
+- `period` is the time in seconds between randomness beacon rounds
+- `genesis_time` is the time in seconds since the Unix Epoch that the group began generating randomness
+- `hash` is the _chain hash_, a hash of the other fields used to uniquely identify the chain of randomness that is being generated by the drand group
```

Comment:
```suggestion
- `hash` is the _chain hash_, which uniquely identifies the drand group. It is used as a root of trust for validation of the first round of randomness.
```

Reply:
It identifies the chain not the group though since group members can change?

Reply:
either chain or "initial group" are fine :)